packages/components/src/components/button/button.component.ts

```diff
@@ -6,29 +6,88 @@ import { BUTTON_COLORS, BUTTON_TYPE, BUTTON_VARIANTS, DEFAULTS } from './button.
 import { getIconSize, isValidIconSize, isValidPillSize } from './button.utils';
 
 /**
- * button component, which ...
+ * `mdc-button` is a versatile component that can be configured in various ways to suit different use cases.
```

Comment:
docs: where are we getting this "is a versatile component" string from? like i am seeing this in all of our components and it seems very copy paste - like can we write a line here which actually describes what the button is?